.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+## What?
+
+Here goes your feature/bug-fix proposal
+
+### Type of change?
+
+- [ ] [build]: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- [ ] [ci]: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+- [ ] [docs]: Documentation only changes
+- [ ] [feat]: A new feature
+- [ ] [fix]: A bug fix
+- [ ] [perf]: A code change that improves performance
+- [ ] [refactor]: A code change that neither fixes a bug nor adds a feature
+- [ ] [style]: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- [ ] [test]: Adding missing tests or correcting existing tests
+
+Note: Remove the non-checked options
+
+## Why?
+
+Here you express the reason it is necessary (keep the _e daí?_ factor in mind)
```

Comment:
```suggestion
Here you express the reason it is necessary (keep the _so what?_ factor in mind)
```
@esequielvirtuoso what do you think?

Reply:
Better than "_e daí?_"!
I'm changing this one.

Reply:
_e dai_ is the best haha just to keep the English pattern